cmd/flow_capture.go

```diff
@@ -147,16 +147,20 @@ func runFlowCaptureOnAddr(port int, filename string) {
 		// terminate capture if max bytes reached
 		totalBytes = totalBytes + int64(bytes)
 		if totalBytes > maxBytes {
-			log.Infof("Capture reached %s, exiting now...", sizestr.ToString(maxBytes))
-			return
+			if exit := onLimitReached(); exit {
+				log.Infof("Capture reached %s, exiting now...", sizestr.ToString(maxBytes))
+				return
+			}
 		}
 
 		// terminate capture if max time reached
 		now := currentTime()
 		duration := now.Sub(startupTime)
 		if int(duration) > int(maxTime) {
-			log.Infof("Capture reached %s, exiting now...", maxTime)
-			return
+			if exit := onLimitReached(); exit {
+				log.Infof("Capture reached %s, exiting now...", sizestr.ToString(maxBytes))
```

Comment:
should be maxTime here?

Reply:
This been addressed in previous PRs :+1: